### DISCUSSION TITLE
Does this plugin support Multicast?

### DISCUSSION THREAD

**Discussion by afedericojr** [DistroAV/DistroAV#737] [Q&A - Support]
It appears that even with decoders set to multicast with a multicast enabled switch, the OBS NDI plugin is still providing duplicate streams for each endpoint, rather than a single broadcast.  Is there a way to enable multicast?

**Comment by nettings** [comment]
I'd be interested in using multicast as well. As per the NDI white paper, it is disabled by default: https://docs.ndi.video/docs/white-paper/10.-multicast. It looks like obs-ndi does not implement the usual JSON-based NDI configuration...

**Comment by RayneYoruka** [comment]
I'm genuinely curious as well, I wonder if enabling it via NDI tools if it can work for obs-ndi

**Comment by Trouffman** [comment]
The plugin does not (yet) use multicast. So if you pull the feed 3 times, it will be 3 times the bandwidth.

This might be integrated in future versions.
